x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/amazonbedrock/request/completion/AmazonBedrockConverseUtils.java

```diff
@@ -7,28 +7,305 @@
 
 package org.elasticsearch.xpack.inference.services.amazonbedrock.request.completion;
 
+import software.amazon.awssdk.core.document.Document;
```

Comment:
Please review the changes in this class carefully, it's new content after the other PR.